### ISSUE TITLE
Usage with rust-bindgen

### ISSUE THREAD

**Issue by Voidoz** [MaikKlein/unreal-rust#28]
Was thinking it might be possible to use the rust-bindgen package to generate ffi instead of writing it manually... Thoughts?

**Comment by gedw99** [comment]
I was thinking the exact thing ..

is more sustainable

**Comment by gedw99** [comment]
All the hard work has been done by @MaikKlein 

the code gen just needs to shim out the code and then tap into it

**Comment by MaikKlein** [comment]
Yep that is quite important for the future of this project see also https://github.com/MaikKlein/unreal-rust/issues/22

**Comment by gedw99** [comment]
Is the goal to create language bindings . For ks, golang , etc ?

if yes then I have done it for golang if that helps?
